### DISCUSSION TITLE
Reuse the preloaded model using cli.

### DISCUSSION THREAD

**Discussion by benjieperez** [openai/whisper#878] [Q&A]
I was wondering if possible to load the model first then reuse it in the entire process via terminal, this is to reduce the processing time for each processing since I'm using multiprocessing.

Thanks.

**Comment by glangford** [comment]
You can give whisper multiple files at once on the command line, or wildcard (eg. `*.mp3`). 

See the discussion and different options here; there is no separate command to load the model in memory.
* https://github.com/openai/whisper/discussions/153#discussioncomment-3888836
